README.md

```diff
@@ -69,103 +69,6 @@ To get a more in-depth introduction, read the [language document](language.md).
 We are currently implementing a first version of Candy in Rust.
 We already have a language server that provides some tooling.
 
-## Long-term TODOs
-
-- Core
-  - io
-  - random
-  - timing
-  - environment variables
-  - HTTP, UDP
-- compiler
-  - make functions independent of their order in top-level scope
-  - patterns
-  - improve pattern match panic messages: `[Foo, 1, {a}] = [Foo, 2, {A: B]]` could generate a message like `` Expected `[_, 1, _]`, got `[_, 2, _]`. ``
-  - "type" proofs
-  - fuzzing of the compiler itself
-  - package root marker
-  - package path dependencies
-  - LLVM, WASM
-- VM
-  - multithreading
-  - object deduplication
-  - profiler
-  - memory representation
-    - inlining of ints/etc.
-    - size of an object
-    - heap management
-- IDE support:
-  - generate debug files
-  - DAP (debug adapter protocol)
-  - [ ] completion, completion resolve
-  - [ ] hover
-  - [ ] signatureHelp
-  - [x] ~~declaration~~, definition, ~~typeDefinition~~
-  - [ ] implementation
-  - [x] references
-  - [x] documentHighlight
-  - [ ] documentSymbol
-  - [ ] codeAction, codeAction resolve
-  - [ ] codeLens, codeLens resolve, codeLens refresh
-  - [ ] documentLink, documentLink resolve
-  - [x] ~~documentColor, colorPresentation~~
-  - [ ] formatting
-  - [ ] rangeFormatting
-  - [ ] onTypeFormatting
-  - [ ] rename, prepareRename
-  - [x] foldingRange
-  - [ ] selectionRange
-  - [ ] prepareCallHierarchy
-  - [ ] callHierarchy incoming, callHierarchy outgoing
-  - [x] semantic tokens
-  - [x] ~~linkedEditingRange~~
-  - [ ] moniker
-- packages
-  - stdin/out utilities such as a print method
-  - files
-  - logging
-  - HTTP Server
-  - Markdown
-  - custom binary serialization
-  - Cap'n Proto
-  - DateTime?
-  - SI?
-  - MongoDB?
-  - package manager
-- online playground
-- clean up repo (delete a bunch of stuff!)
-
-## Short-term TODOs
-
-- new name?
-- add caching while compile-time evaluating code
-- tags
-- add tests
-- add a more lightweight tracer that only tracks stack traces
```

Comment:
Is it intentional that there's no issue for this one, or did I overlook it? (The rest looks good)